src/overview/utils/helpers/OverviewUserSettings.ts

```diff
@@ -9,26 +9,29 @@ type OverviewUserSettings = {
 };
 
 type WelcomeSettings = {
-  hideWelcome: boolean;
+  hideWelcome?: boolean;
   save: (showWelcome: boolean) => void;
   clear: () => void;
 };
 
-const parseOrClean = (key) => {
+// eslint-disable-next-line @typescript-eslint/no-unnecessary-type-parameters
+const parseOrClean = <T>(key: string): T | object => {
   try {
-    return JSON.parse(loadFromLocalStorage(key)) ?? {};
+    const storedValue = loadFromLocalStorage(key) ?? '';
+    return JSON.parse(storedValue) as T;
   } catch (_e) {
     removeFromLocalStorage(key);
+    // eslint-disable-next-line no-console
```

Comment:
Can you expose in a shared util high in the hierarchy export const MTVConsole = console , then u can use it as console log without comments, as it intends to log that aren't forgotten